feature/feature.go

```diff
@@ -35,15 +35,6 @@ const (
 	// beta: v1.7
 	MachinePool featuregate.Feature = "MachinePool"
```

Comment:
Let's also:
* Remove `,ClusterResourceSet=${EXP_CLUSTER_RESOURCE_SET:=true}` from config/manager/manager.yaml
* Remove l.6-l.11 in docs/book/src/tasks/cluster-resource-set.md